### DISCUSSION TITLE
...suggestions ?

### DISCUSSION THREAD

**Discussion by MarcoRavich** [MarcoRavich/Opendows#1] [Ideas]
Please feel free to post any open source replacement you know/find here !

**Comment by V4nnura** [comment]
1. LegacyUpdate is much better than WuMgr and LegacyUpdate it's still updating, i think is better change it. Here: https://github.com/kirb/LegacyUpdate
2. WineVDM is a best NTVDM replacement i found about. Here: https://github.com/otya128/winevdm
3. WineD3D for Windows is a DirectX 1-11 to OpenGL wrapper based on WineD3D. Here: https://github.com/adolfintel/wined3d4win
4. To a Media Player replacement i think the Media Player Classic or the VLC media player.
5. And NTFS-3G it's the best open source NTFS drive that i can found it. Here: https://github.com/tuxera/ntfs-3g

**Reply by V4nnura** [reply]
If you want to know more something, talk to me.

**Reply by V4nnura** [reply]
An addendum: The cmder site is not the same anymore, and i think the best it's to direct the repository not the website itself, so: 
https://github.com/cmderdev/cmder
https://github.com/Feodor2/Mypal68

**Comment by MarcoRavich** [comment]
Hi there @Leandro2905, 1st of sorry for the late reply (we're a bit busy with our other project [HyMPS](http://www.forart.it/progetti/HyMPS/)) and thanks for all suggestions.

We've [actively supported](https://reactos.org/forum/search.php?keywords=forart) [ReactOS](https://reactos.org/) in the past, so we already know many of them.

The Opendows concept was inspired by the early release of @xeno86's [KernelEx](https://kernelex.sourceforge.net/) and our long-standing attendance (started in 2005) of the [MSFN\Windows 9x Member Projects forum](https://msfn.org/board/forum/91-windows-9x-member-projects/): why not apply the [GNU approach](https://www.gnu.org/gnu/initial-announcement.html) to Windows too ?

The goal would therefore be to release a "patcher tool" (something like @wyattoday's [wyUpdate](https://github.com/wyday/wyupdate#wyupdate-the-open-source-updater-program-for-windows) or - more contemporary - @builtbybel's [ThisIsWin11](https://github.com/builtbybel/ThisIsWin11/)) that could have multiple advantages:
1. provide updates for EOL Windows versions;
2. installed system functions continuity;
3. avoiding licensing issues;
4. promote open source softwares usage among users.

It might be also useful for distributing official updates that are no longer available through MS channels (such as those collected by other alternative projects like @kirb's [Legacy Update](https://legacyupdate.net/) for example), but the target is to gradually replace all components of the installed operating system.

Last but not least, feel free to update/mod/add resources to Opendows by directly github project pages !

Thanks again.

**Reply by V4nnura** [reply]
Ok, thanks for your response @forart , i just want to contribute the project because of their resembleance to one of my objectives projects: Replacing some programs to open-source counterparts. I created a fork now to help more to add somethings. Still remember this project, this objective is helpful and potential!

**Comment by MarcoRavich** [comment]
Hi @Leandro2905, after some time I'm back in this discussion: just added @ntdevlabs's [tiny11builder](https://github.com/ntdevlabs/tiny11builder) in the [notebloc page](https://github.com/forart/Opendows/blob/main/note.md) that sounds quite inspiring for Opendows' "strategies" too...

**Reply by V4nnura** [reply]
Good to know =D

**Comment by codehacker47** [comment]
Maybe instead of putting all of this open source software in some lists, make a group of some developers in these lists to collaborate and create a user-friendly "patcher tool" for making Windows a great OS.

**Reply by MarcoRavich** [reply]
Yes, that's the idea.

Anyway I'm not a developer...
